FredLin.md

```diff
@@ -1,168 +1,17 @@
 [Open bugs assigned to me](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=assignee%3Agasolin%40mozilla.com) (ASSIGNED = current working on; NEW = backlog)
 
-## 08/07 ~ 09/11
```

Comment:
@gasolin I am sorry, but you still need a weekly section on top of the Q sections.